automl/automl-library/library.whizzml

```diff
@@ -83,19 +83,23 @@
     (raise (str "If you don't define a correct train-dataset,"
                 "you must define a correct previous execution"))))
 
-;; Checks excluded fields. We can't reuse resources from
-;; previous executions if the excluded_fields were different.
-(define (check-excluded-fields train exec excluded)
-  (if (and (not (= (resource-type train) "dataset"))
-           (= (resource-type exec) "execution")
-           (not (empty? excluded))
-           (not (= excluded (input-from-exec exec "excluded_fields"))))
-    (raise (str "Your excluded_fields are different to the ones you provided"
-                " in the given previous execution of automl."
-                " Please, provide the same excluded_fields (or leave blank the"
-                " excluded_fields input to use the same excluded_fields as in"
-                " the training execution) or provide a training dataset to start"
-                " training again with the new excluded_fields"))))
+;; Checks that all the needed params are in the configuration
+(define (check-configuration-params config)
+  (let (correct #["excluded-fields" "excluded-models"
+                  "pca-variance-threshold" "leverage-threshold"]
```

Comment:
leverage-threshold applies to lift too? maybe association-threshold or rule-strenght-threshold?...

Reply:
Leverage and lift take different range of values. Currently I am only filtering by leverage because is the only parameter that has defined values range. Lift can't take all real values. I still create two associations ordering by lift and leverage but I filter both list of rules by leverage. It's not the perfect solution but I can't imagine a better one.

Reply:
Anyways, I could change the name as you are suggesting, providing in that way more abstraction.